pkg/reconciler/gc/v2/gc.go

```diff
@@ -61,6 +61,9 @@ func Collect(
 	})
 
 	for _, rev := range revs[gcSkipOffset:] {
+		if _, ok := rev.ObjectMeta.Annotations[serving.RevisionPreservedAnnotationKey]; ok {
```

Comment:
Should we check that value is non-empty? I guess the name (as it is) asks for a `bool` value. So preserve=false, means GC is possible.

Reply:
Done.